tajo-catalog/tajo-catalog-client/src/main/proto/CatalogProtocol.proto

```diff
@@ -121,9 +121,12 @@ service CatalogProtocolService {
   rpc existPartitionMethod(TableIdentifierProto) returns (ReturnState);
 
   rpc getPartitionByPartitionName(PartitionIdentifierProto) returns (GetPartitionDescResponse);
-  rpc getPartitionsByTableName(PartitionIdentifierProto) returns (GetPartitionsResponse);
+  rpc getPartitionsByTableName(TableIdentifierProto) returns (GetPartitionsResponse);
+  rpc existPartitionsByTableName(TableIdentifierProto) returns (ReturnState);
```

Comment:
As I mentioned, ``isPartitionedTable`` would be better. It may not need the suffix ``byPartitionName``.